src/pages/publication/SubmissionPanel.tsx

```diff
@@ -54,6 +55,16 @@ const SubmissionPanel: React.FC<SubmissionPanelProps> = ({ savePublication }) =>
         </Card>
         <Card>
           <SubHeading>{t('heading.references')}</SubHeading>
+          <LabelContentRow label={t('common:type')}>
+            {publicationType && t(`referenceTypes:${publicationType}`)}
+          </LabelContentRow>
+          <LabelContentRow label={t('publication.link_to_publication')}>
+            {doiUrl && (
+              <Link href={doiUrl} target="_blank" rel="noopener noreferrer">
+                {doiUrl}
+              </Link>
+            )}
+          </LabelContentRow>
```

Comment:
Skal LabelContentRow rendres hvis man ikke har publicationType eller doiUrl? Bedre å flytte sjekken utenfor LabelContentRow?

Reply:
nå viser vi alle felter selv om de ikke har verdi for å gi brukeren et hint om hvor det mangler input

Reply:
ahh, skjønner

Reply:
når jeg tenker meg om så skal ikke brukeren få lov til å skrive i doi-feltet i etterkant, så akkurat her gir det mening å droppe det helt. good catch! 😄